mods/ts/rules/structures.yaml

```diff
@@ -127,6 +127,31 @@ GAPOWRUP:
 	Power:
 		Amount: 50
 
+GASPOT:
+	Inherits: ^Building
+	Buildable:
```

Comment:
I don't think it's a good idea to base our choices of what is and isn't buildable exclusively on the unused assets that WW forgot to remove from the mixes.  These don't perform any useful purpose except for map decoration, so should be left to the map editor.